packages/playground-examples/scripts/generateTOC.js

```diff
@@ -96,7 +96,7 @@ langs.forEach(lang => {
         .replace(/\+/g, "-"),
 
       sortIndex: index,
-      hash: crypto.createHash("md5").update(contents).digest("hex"),
+      hash: crypto.createHash("sha512").update(contents).digest("hex"),
```

Comment:
Checking this, the website still works fine here. It's put into `data-hash` in `ShowExamples` and that's just used to cache loading them.